domains/kishore.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "personal portfolio",
+    "repo": "https://github.com/Kishore0122/portfolio",
+    "owner": {
+        "username": "Kishore0122",
+        "email": "kishorepanuganti278@gmail.com"
+    },
+    "record": {
+        "CNAME": "kishore0122.github.io/portfolio/"
```

CNAME can't contain `/`
```suggestion
        "CNAME": "kishore0122.github.io"
```